### ISSUE TITLE
How to recover serverafter rear backup is moved to a different location

### ISSUE THREAD

**Issue by dheeraj86mathur** [rear/rear#990]
# Relax-and-Recover (rear) Issue Template

Please fill in the following items before submitting a new issue:

* rear version (/usr/sbin/rear -V): 1.18
* OS version (cat /etc/rear/os.conf or lsb_release  -a): RHEL 6.7
* rear configuration files (cat /etc/rear/site.conf or cat /etc/rear/local.conf):
OUTPUT=ISO
BACKUP=NETFS
BACKUP_URL="file:///var/rear"
OUTPUT_URL="file:///var/rear"
NETFS_KEEP_OLD_BACKUP_COPY=y
BACKUP_PROG_EXCLUDE=("${BACKUP_PROG_EXCLUDE[@]}" '/mnt' '/media' '/var/crash')

* Brief description of the issue
I got rear generated on local FS /var/rear and then copied it to a NFS server manually, as the server is behind firewall. Now when I boot the server with rear iso recovery, how do I specify path for NFS share(new location of rear) when I am at relax-and-recover menu.

* Work-around, if any

**Comment by gdha** [comment]
Just edit the `local.conf` file and change the variable content of `BACKUP_URL` to the new location. Hopefully, you are not missing any NFS related executables (as these may not be copied during _mkbackup_ phase as it was not required).  You can try and verify it...

**Comment by dheeraj86mathur** [comment]
I think I did not clarified issue properly. Server is behind firewall. NFS will not work. So, I have intentionally used BACKUP_URL=file:/// option in local.conf. Now, I have rear generated on local FS, and I have copied it on a  jumphost for this firewalled server. If I boot the server with rear-server.iso file generated with rear mkbackup, It tries to recover from BACKUP_URL=file:///localdirectorypath, which it cannot access at time of recovery. So, now how can modify this parameter (after rear is already generated and server is waiting to recovered), when I have changed the local of my rear backup.

**Comment by gdha** [comment]
@dheeraj86mathur well guess you answered your own question. Just change the BACKUP_URL to the new location before starting `rear -v recover` will do the trick

**Comment by dheeraj86mathur** [comment]
That brings in my exact question, where can I modify BACKUP_URL to new location, because start rear -v recover ?? I cant modify local.conf now, as the server is already down.  Even if i could modify local.conf, it will not update the existing rear backup and iso, which I have already generated. One option is to mount iso file in rw mode and then modify BACKUP_URL value in isolinux.cfg .... But will it work, I havent tried this yet and want to know if that is correct method ..

**Comment by gdha** [comment]
@dheeraj86mathur The complete rear environment is still available in the rescue image. Just edit `/etc/rear/local.conf` and you are ready for the recover ;-)

**Comment by jsmeix** [comment]
As far as I understand it the issue is sufficiently answered
so that I close it, if needed it can be reopened.
